### PR TITLE
add task-spooler script

### PR DESCRIPTION
**Pull request by shanedabes**
Script to show queued/running count for one or more task spooler servers.

### CODE REVIEW

**Comment by shanedonohoe** [comment]
Wow, shellcheck's posix compliance is amazingly restrictive. I'll work on some (probably awful) compliant way of getting past this test failure tomorrow.

**Comment by shanedonohoe** [comment]
I guess that will do it :sunglasses:.

File: polybar-scripts/info-task-spooler/info-task-spooler.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```
Comment (x70b1) [inline polybar-scripts/info-task-spooler/info-task-spooler.sh]: Lets use #!/bin/sh in this case.

**Comment by shanedonohoe** [comment]
Ok sure, I will do, but why is that necessary? This is a bash script.

**Comment by x70b1** [comment]
Ha, you are wrong! :sunglasses: 
It is a shell script. And this means there is no need why to force a specific shell :wink: 

Thanks!

**Comment by shanedonohoe** [comment]
@x70b1 how does it look now?

**Comment by shanedabes** [comment]
@x70b1 are there any problems left?

**Comment by x70b1** [comment]
@shanedabes Sorry for that delay. 

It looks good, I will merge it.
But I need some time to check everything. I hope I get this done in the next days.

Thank you for your work! :+1:

**Comment by x70b1** [comment]
I merged it. Sorry for the delay.

**Comment by shanedabes** [comment]
Great!

Looks like the screenshot was deleted though?

**Comment by x70b1** [comment]
I renamed the directory but cant rename the screenshot. 
Thats why i deleted it and now he is back and on our main `README.md`.

**Comment by shanedabes** [comment]
Yep, I see it's back now.

Looks like we're all good, expect to have more contributions from me in the future, then we can go through this all again :)

**Comment by x70b1** [comment]
We could refactor your script a bit. My idea would be an exec like:
`info-taskspooler.sh --socket yt,youtube p,podcast`

Your script would start something like

```
case "$1" in
    --socket)
         taskspooler_sockets
  
        ;;
    *)
         taskspooler_count

        ;;
esac
``

**Pull request merged**